### ISSUE TITLE
Accessibility: [MAS4.3.1]: The icon above 'Report issues' link is not visible in high contrast white mode.

### ISSUE THREAD

**Issue by tylerbutler** [microsoft/FluidFramework#3812]
**Build details:**

**OS Version:** 2004(20221.1000)

**Edge Dev Version**: 87.0.654.0 (Official build) dev (64-bit)

**URL:** <https://fluidframework.com/community/>
  

**Pre-requisite:**

Turn on High contrast mode.(Settings->Ease of access->High Contrast->Choose a theme->High Contrast Black/White)  
  

**Repro Steps:**

1\. Navigate to 'Report issues' link.

2.Now observe the icon above that link.

  
**Actual Result:**  

The icon above 'Report issues' link is not visible in high contrast white mode.  

**Expected Result:**  
The icon above 'Report issues' link should be visible in high contrast white mode.  

  
**User Impact:**  

Users with low vision will not able to consume the icon if it is not visible at all in high contrast white mode.

  

**MAS Reference:**

**MAS 4.3.1 - No Disruption of Accessibility Features**
